### ISSUE TITLE
Unable to connect to my Gmail account on Franz

### ISSUE THREAD

**Issue by maxleprof** [meetfranz/franz#1584]
Dear all,

I am unable to connect to Gmail on Franz whatever the laptop (I tried on a few).
When I enter my ID, then my password, this message appears : 
_You are trying to sign in from a browser or app that doesn't allow us to keep your account secure. Try using a different browser. Learn more_
When I try with another account, there is no issue.

How to solve it ?

Thank you so much for your help & answers !

Best,

**Comment by anglisc** [comment]
I have found the same issue with Google calendar too.

**Comment by maxleprof** [comment]
> I have found the same issue with Google calendar too.

@anglisc : for how long have you met the issue ? No solution, I guess ?

**Comment by eugeneromero** [comment]
This is a duplicate of #1573, still no word from the devs...

**Comment by maxleprof** [comment]
Hello @eugeneromero ,
Funny that you answered the post.
I just tried to connect my Gmail account again and it seems that everything is alright.
I think that this is a matter of time ...
In my situation, I have been blocked around two weeks.
Good luck !
Best,

**Comment by eugeneromero** [comment]
Hey @maxleprof, yep, after reading your comment I can confirm it now works again... very odd indeed. Lets see how long this lasts :stuck_out_tongue:

**Comment by anglisc** [comment]
I noticed it about 7 days ago when I commented on the issue. I can confirm that google services are letting me log in again.

I noticed after I changed my password it wouldnt let me authenticate with the error message reported above by @maxleprof.

**Comment by DasJott** [comment]
Nope. I only get the register/login page. If I click on login, Browser opens and Franz stays  on the page.
No way to use Gmail from within Franz currently.

**Comment by yaser-alazem** [comment]
Still not logging on Gmail! Hope to solve this issue.

**Comment by fjandrad** [comment]
Dunno if it was solved in the past, but it is complaining about the browser considered to be unsafe. SO can not connect gmail.

**Comment by SMile61** [comment]
Same issue. Franz considered as unsafe, no connection on any gmail service

**Comment by asymetricbalance** [comment]
Please see workaround at #1771 or #1800

**Comment by SMile61** [comment]
> Please see workaround at #1771 or #1800

Doesn't work anymore (W10, 2 factor authentication disabled)
After modifying the file, The recipe loads a "create account page":
![image](https://user-images.githubusercontent.com/62472410/77736079-02603780-700c-11ea-8cda-2a8eec639c3b.png)
Any clic (connexion, create account,...) loads in the navigator and not inside Ferdi

**Comment by asymetricbalance** [comment]
SMile61 - I just added one gmail account to Franz using a modified index.js file in
C:\Users\[username]\AppData\Roaming\Franz\recipes\gmail
Worked on Win 10, v1909
Franz v5.4.0
Gmail - no 2FA

Did you restart Franz after making the change?
I was asked to use chrome after the initial set up, but declining that "offer" it went straight to the inbox.
What could be different in your setup?

Attaching my file here as txt.

[index.js.txt](https://github.com/meetfranz/franz/files/4394925/index.js.txt)

**Comment by SMile61** [comment]
@asymetricbalance: Thanks for your support, and please forgive my questions if they are too silly, I'm a complete newbee here

- I reloaded each time I tried a change
- I first thought that I had to change "useragent" with "Gmail", then I tried to directly paste your file with/without the sourcemappingUrl part (what does that stand for?). No change, I load the same page each time.

- Didn't understand your reference to "declining chrome's offer": I d
on't see any possibility to decline on the loaded page

**Comment by asymetricbalance** [comment]
First - There are no silly questions.
Second - this how it goes...
- download the file to your Desktop and rename it from index.js.txt to index.js (remote the "txt" portion.) 
- open the folder C:\[your_username]\AppData\Roaming\Franz\recipes\gmail and rename the index.js file to index.js.bak (in case something does wrong.)
- move or copy the file from the Desktop into that folder.
- quit Franz completely (not only reload)

I just copied the file directly from my folder without any changes. You can remove lines starting with "//" - they are not executed.)

Also - open the index.js.txt file to verify there is no weird code in there. I know I didn't tamper with anything, but always be on the safe side.

**Comment by SMile61** [comment]
Seems that the problem comes from 5.3.4 upgrade, it's finally working after reinstalling ferdi.

What I did exactly:
- using your recipe =>no change
- Move defaut navigator from opera to google =>no change
- Uninstalling Franz, just keeping Ferdi => no change
- uninstalling ferdi, then reinstalling 5.3.3 => WORKING
- upgrading Ferdi by clicking on the update banner => Not working any more
- uninstalling ferdi, then reinstalling 5.3.3 => WORKING again
So I stay on 5.3.3. for the moment ;-)

Any chance to hide the update banner?

**Comment by SMile61** [comment]
update my post
looks like it's not an update from Ferdi but from Franz. Maybe I didn't correctly move my account?
![image](https://user-images.githubusercontent.com/62472410/77818435-7f55e480-70d2-11ea-8c59-cc22a1a2ffb2.png)

**Comment by asymetricbalance** [comment]
I am using 5.4.0, but always update by downloading from the web page, not using the internal upadte function. Is it possible that the upgrade overwrites the index.js file?

**Comment by SMile61** [comment]
I first tried Franz 5.4.0 but didn't manage to use Gmail, then went to Ferdi wich whas 5.3.3 by the internal update button, and didn't manage better. The update banner simply loads a 5.3.4 release version, I didn't completely pay attention to the changes but I'm pretty sure that the index.js file wasn't changed.

Anyway, It looks like the latest releases are 5.4.1 on Franz and 5.4.3 on Ferdi. I'll try the 5.4 later

Thanks a lot for your help

**Comment by seyhajin** [comment]
> > Please see workaround at #1771 or #1800
> 
> Doesn't work anymore (W10, 2 factor authentication disabled)
> After modifying the file, The recipe loads a "create account page":
> ![image](https://user-images.githubusercontent.com/62472410/77736079-02603780-700c-11ea-8cda-2a8eec639c3b.png)
> Any clic (connexion, create account,...) loads in the navigator and not inside Ferdi

I've find a solution that worked for me:
1. Open Franz app
2. Find Gmail service ID : activate devtools and search `partition` attribute  (ex: `partition="persist:service-a7f0a998-be50-4f61-a18b-002f19df120f"`) of webview tag `is-active`.
3. Close Franz app
4. Go to Franz Partitions folder (on Windows: %APPDATA%\Franz\Partitions), backup first before delete folder corresponding of service ID (ex: `\service-a7f0a998-be50-4f61-a18b-002f19df120f`)
5. Restart Franz
6. Connect to gmail
7. That's all :)

**Comment by jmandt** [comment]
Hi there, 

I am using macOS Mojave Version 10.14.6 and having the same problem. Any idea on how to fix it on Mac? I already messaged the support a couple of times but I did not receive any answer even though I am on a paid plan. -.-

Any help would be very appreciated

**Comment by riprop** [comment]
Problem not solved after more than a year !!!

Franz Team please come with solution or announce officially Franz is not supporting gmail anymore.

P.S. Everyone tying to resolve problem with file-editing-hacks can not be called a solution !!!

Thank you !

**Comment by jmandt** [comment]
@riprop Propbem seems to be solved. Solution for me was to install the newest update of Franz.
